homeassistant/components/smartthings/strings.json

```diff
@@ -17,11 +17,20 @@
         "description": "Please select the SmartThings Location you wish to add to Home Assistant. We will then open a new window and ask you to login and authorize installation of the Home Assistant integration into the selected location.",
         "data": { "location_id": "[%key:common::config_flow::data::location%]" }
       },
-      "authorize": { "title": "Authorize Home Assistant" }
+      "authorize": { "title": "Authorize Home Assistant" },
+      "reauth_confirm": {
+        "title": "Reauthorize Home Assistant",
+        "description": "You are about to reauthorize Home Assistant with SmartThings. This will require you to log in and authorize the integration again."
+      },
+      "update_confirm": {
+        "title": "Finish reauthentication",
+        "description": "You have almost successfully reauthorized Home Assistant with SmartThings. Please press the button down below to finish the process."
```

Comment:
Why do users need to press this button? Can't we just finish it?

Reply:
Because we're receive the tokens via a webhook call, and then do async_configure on that flow. Downside of this is they if that async configure finishes the flow, the user is greeted with a "invalid flow message" because the old flow has been removed. So this extra step make sure it's a bit more user friendly